oxford-metadata.ttl

```diff
@@ -334,6 +334,13 @@
     rdfs:label "Markov Model State" ;
     rdfs:comment "For any state variable which is a state in a Markov model for an ion channel." .
 
+:not_a_probability_even_though_it_should_be
+    a :Annotation ;
+    a :MultipleUsesAllowed ;
+    a :Probability ;
+    rdfs:label "Probability that does not stay within 0 ... 1 range" ;
+    rdfs:comment "This should be a probability but it is not guaranteed to stay within 0 ... 1 range." .
```

Comment:
```suggestion
    rdfs:comment "This should be a probability but it is not guaranteed to stay within [0,1] because of unusual mathematical formulation. Introduced to prevent checks being generated in code for quantities annotated with this tag" .
```